docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-alpine3.8
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+ENV AWS_CONFIG_FILE=/root/.aws/config
+ENV NEO4J_HOST=neo4j
+
+ADD https://raw.githubusercontent.com/eficode/wait-for/master/wait-for .
```

Comment:
Is there a way to add this same functionality without adding in code from an external repo?

Reply:
Maybe if I download the script and add it to the repository. Do you think that's acceptable?

Reply:
@achantavy  I did just that: downloaded the script, and added it to the cartography repository. It's in the newest commit.